pkg/controller/statefulset/stateful_set.go

```diff
@@ -52,6 +52,10 @@ var controllerKind = apps.SchemeGroupVersion.WithKind("StatefulSet")
 // podKind contains the schema.GroupVersionKind for pods.
 var podKind = v1.SchemeGroupVersion.WithKind("Pod")
 
+const StatefulSetIndex = "statefulSetName"
+
+var OrphanPodIndexKey = "Orphan"
```

Comment:
Maybe consider something that is more unlikely to be used by user like "_ORPHAN_POD"

Reply:
+1